.gitignore

```diff
@@ -68,5 +68,3 @@ Cargo.lock
 .history
 
 # End of https://www.gitignore.io/api/git,rust,macos,visualstudiocode
-
-# !Cargo.lock
```

Comment:
Uncomment this line to allow the commiting of Cargo.lock

Reply:
Done